core-services/prow/02_config/_config.yaml

```diff
@@ -643,6 +643,7 @@ tide:
     - release-4.3
     - release-4.5
     - release-4.6
+    - release-4.7
```

Comment:
Why did we have to do this in this PR? Was there not a branching PR that did this? @droslean